postgreSQL-case-study.md

```diff
@@ -0,0 +1,172 @@
+
+## Weaknesses
+
+### Learning curve
+Learning PostgreSQL can be a little hard, and it can take time to be good at it. Installations, configurations, and such can be fairly difficult for beginners.
+
+### Performance
+Even though PostgreSQL is a very efficient database, it may not be as fast as some other databases in certain areas.
+
+### Support
+Compared to MySQL where many open-source apps can support it, PostgreSQL may not be.
+
+## Comparison
+
+### PostgreSQL VS MYSQL
+
+#### Differences
+* PostgreSQL is an Object Relational Database Management System(ORDBMS) and MySQL is Relational Database Management System(RDBMS). Meaning in MySQL, all tables are interrelated, but PostgreSQL combines the characteristics of both ORDBMS and RDMBS so it doesn't only contain interrelated tables, but it also supports objects, classes, and inheritances.
+* PostgreSQL supports modern application features such as JSON, XML etc, while MySQL only supports JSON.
+* While PostgreSQL supports advanced data types such as arrays, hstore, and user-defined data types, MySQL only supports standard data types such as numeric, date and time, string, spatial and the JSON data type.
+* PostgreSQL is maintained by developers who volunteer, while MySQL is maintained by Oracle Corporation.
+* PostgreSQL is also fully ACID compliant while MySQL is ACID compliant only through InnoDB and NDB Cluster Storage engines. 
+* In terms of how the learning curve is for these databases, PostgreSQL may be harder for beginners to learn compared to MySQL which is more user-friendly as takes less time to build a project from scratch.
+
+#### Similarities
+* Both rely on SQL which enables tables to be joined using few lines of code.
+* Both PostgreSQL and MySQL have communities to provide support to their users where they have many volunteers that devote their time to offer free advice, recommendation and support to users.
+* On both DBMSs, you can use JSON for transporting and storing data, objects, and arrays.
+
+## Summary
+
+PostgreSQL, also known as Postgres, is a free and open-source Object Relational Database Management system with more than 30+ years of active development. It dates back to '80s when it was created by a computer science professor named Michael Stonebraker at the University of California, Berkley. It has then been renamed twice and now called PostgreSQL to signify the relationship between the original name and the use of SQL. It has many advanced features that safely store and scale the most complicated workloads. Though PostgreSQL has its strength with its many built-in features and more such as supporting many programming languages, being extensible, and handling large amount of data, it also has its challenges. PostgreSQL has a steep learning curve, making it hard for beginners and it also may be slower than other databases in certain workloads. 
```

Comment:
```suggestion
PostgreSQL, also known as Postgres, is a free and open-source Object Relational Database Management system with more than 30+ years of active development. It dates back to the 80s when it was created by a computer science professor named Michael Stonebraker at the University of California, Berkley. It has since been renamed twice and is now called PostgreSQL to signify the relationship between the original name and the use of SQL. It has many advanced features that safely store and scale the most complicated workloads. Though PostgreSQL has its strength with its many built-in features and more such as supporting many programming languages, being extensible, and handling large amounts of data, it also has its challenges. PostgreSQL has a steep learning curve, making it hard for beginners, and it also may be slower than other databases in certain circumstances. 
```